src/pages/projectSettings/CreateProjectModal.tsx

```diff
@@ -38,6 +41,11 @@ export const CreateProjectModal: React.VFC<Props> = ({
   });
   const [hasError, setHasError] = useState(true);
 
+  const { data } = useQuery<GetGithubOrgsQuery>(GET_GITHUB_ORGS);
```

Comment:
adding a `{skip: !open }` to the params would prevent this query from being kicked off if the user never opens this modal.